scripts/run-e2e.ts

```diff
@@ -137,30 +135,6 @@ const initStorybook = async ({ cwd, autoDetect = true, name }: Options) => {
   }
 };
 
-// Verdaccio doesn't resolve *
-// So we set resolutions manually in package.json
-const setResolutions = async ({ cwd }: Options) => {
```

Comment:
I was going to ask about this one. Does it make sense to resolve to sibling path in order to use local sb when working on e2e ?
This would also help getting examples out of the monorepo.
Does this even work !?